src/components/AddBook.js

```diff
@@ -1,8 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { bookAdded } from '../redux/books/books';
 
 const AddBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  // const [author, setAuthor] = useState('');
```

Comment:
- [ ] Kindly remove commented code. It is often not a good practice as it makes your code looks untidy and sometimes unreadable